### PR TITLE
Add new tlv's for vxlan and dcbx feature vfp entries

### PR DESCRIPTION
**Pull request by harshsin**
Reviewer: @rlane 
cc @poolakiran @nick-bsn

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req/228/

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -756,3 +756,14 @@ struct of_bsn_tlv_hash_gtp_port_match : of_bsn_tlv {
...
+
+struct of_bsn_tlv_untagged : of_bsn_tlv {
+    uint16_t type == 106;
+    uint16_t length;
+};
+
+struct of_bsn_tlv_vfp_class_id : of_bsn_tlv {
+    uint16_t type == 107;
+    uint16_t length;
+    uint8_t value;
```
Comment (harshsin) [inline openflow_input/bsn_tlv]: changed it to 32.

**Comment by rlane** [comment]
ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2016.03.10.2248-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req/229/

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2016.03.10.2248-m.loxigen.master/abat.log
/cc

**Pull request merged**